docs/usage.rst

```diff
@@ -123,12 +177,16 @@ are intended to validate older or pre-deployed servers, although with slightly m
 A basic `Magpie` instance can also be initialized by using the `docker-compose.yml.example`_ file.
 Note that the environment variables :envvar:`MAGPIE_TEST_REMOTE_SERVER_URL` and :envvar:`HOST_FQDN`
 must first be defined for the `remote` tests.
-For example, these default values should work with ``docker-compose``::
+For example, these default values should work with ``docker-compose``:
+
+.. code-block:: console
 
     export MAGPIE_TEST_REMOTE_SERVER_URL=http://localhost:2001
```

Comment:
Is it still valid since we also need the MAGPIE_PORT variable?

Reply:
This one is only for tests. 
`MAGPIE_PORT` is needed only if `MAGPIE_URL` or `magpie.url` in the INI was not defined.